skrub/tests/test_interpolation_join.py

```diff
@@ -28,35 +30,34 @@ def weather():
     )
 
 
-@pytest.mark.parametrize("key", [["latitude", "longitude"], "latitude"])
-@pytest.mark.parametrize("with_nulls", [False, True])
-def test_interpolation_join(df_module, buildings, weather, key, with_nulls):
-    if df_module.name == "polars":
-        pytest.xfail(
-            reason=(
-                "In polars, DataFrame.drop() got an unexpected keyword argument 'axis'."
-            )
-        )
-    weather = weather.fillna(0.0)
+def test_simple_join(df_module, buildings, weather):
+    weather = df_module.DataFrame(weather)
```

Comment:
I think we can parametrize the fixtures themselves with `df_module` but i think you planned that for a future PR

Reply:
Yes I will do multiple follow-ups after merging this